test/e2e/edgedevice_test.go

```diff
@@ -255,6 +269,20 @@ func (e *edgeDeviceDocker) Register(cmds ...string) error {
 	})
 }
 
+func (e *edgeDeviceDocker) ValidateNoDataRaceInLogs(extraCommands ...string) (bool, error) {
+	logsMap, err := e.GetLogs(extraCommands...)
+	if err != nil {
+		fmt.Printf("Error: GetLogs failed: %v \n", err)
+		return false, err
+	}
+	for _, output := range logsMap {
+		if strings.Contains(output, "WARNING: DATA RACE") {
```

Comment:
will the race be logged?

Reply:
shouldn't be needed at all TBH, because if test fail, it'll dump all logs :-)